solutions/usaco-834.mdx

```diff
@@ -62,4 +62,31 @@ public class OutOfSorts {
 }
 ```
 </JavaSection>
+
+<PySection>
+
+```python
+import sys
+
+sys.stdin = open("sort.in", "r")
+sys.stdout = open("sort.out", "w")
+
+input = sys.stdin.readline
+
+n, fix = int(input()), 111111
+l, locations = [0] * n, {}
+
+for i in range(n):
+	idx = int(input()) * fix + i
```

Comment:
huh what's `fix` for

Reply:
because "The first line of input contains N (1≤N≤100,000)"

Reply:
um, probably use a named constant then